app/components/embed/footer_component.html.erb

```diff
@@ -28,6 +29,20 @@
       </a>
     <% end %>
 
+    <% if viewer.display_download_all? %>
+      <a id='sul-embed-footer-download-all' href='<%= viewer.download_url %>' target="_blank" rel="noopener noreferrer nofollow" data-controller="download" data-action="click->download#download">
+        <button class='sul-embed-footer-tool btn btn-toolbar sul-i-download-3' aria-label="download all files">
+          Download all <%= pluralize(viewer.purl_object.downloadable_files.length, "file") %>
+          (<%= viewer.pretty_filesize(viewer.purl_object.size) %>)
+          <% if viewer.any_stanford_only_files? %>
+            <span class="sul-embed-location-restricted-text sul-embed-stanford-only-text" aria-label="restricted to stanford">
+              <span class="sul-embed-text-hide"> Stanford only </span>
+            </span>
+          <% end %>
+        </button>
+      </a>
+    <% end %>
+
```

Comment:
This makes the download all link available to any viewer that uses the common `FooterComponent`